pyscreeze/__init__.py

```diff
@@ -524,7 +524,7 @@ def _screenshot_osx(imageFilename=None, region=None):
     TODO
     """
     # TODO - use tmp name for this file.
-    if tuple(PIL__version__) < (6, 2, 1):
```

Comment:
Was this change compatible with any other system before? We don't want to break things that worked @asweigart

Reply:
If by other systems you mean other operating systems then that entire function is macOS only.

Reply:
It literally says "_osx" (os X aka macOS I am on latest Sonoma)
--> def _screenshot_osx(imageFilename=None, region=None): <----
<img width="579" alt="osx" src="https://github.com/asweigart/pyscreeze/assets/34134411/7e729268-8dcf-4fe2-ae1a-9c7c3b55bbf3">

Reply:
> 

only if you wanna support old hardware and ignore new macOS versions, didn't test. Nobody cares really and those who care will just re-write source to accommodate their needs.